<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KanjiBrowserView</class>
 <widget class="QWidget" name="KanjiBrowserView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Kanji Browser</string>
  </property>
  <widget class="QStackedWidget" name="_stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>681</width>
     <height>471</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>681</width>
       <height>471</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0" colspan="3">
       <widget class="KListWidget" name="_kanjiList">
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="dragDropMode">
         <enum>QAbstractItemView::NoDragDrop</enum>
        </property>
        <property name="viewMode">
         <enum>QListView::IconMode</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="KComboBox" name="_grades"/>
      </item>
      <item row="1" column="0">
       <widget class="KComboBox" name="_strokes"/>
      </item>
      <item row="0" column="1" rowspan="2">
       <widget class="KHistoryComboBox" name="_searchBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="2" rowspan="2">
       <widget class="KPushButton" name="_searchButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QTextBrowser" name="_kanjiInformation">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>641</width>
       <height>441</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="overwriteMode">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KListWidget</class>
   <extends>QListWidget</extends>
   <header>klistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KHistoryComboBox</class>
   <extends>KComboBox</extends>
   <header>khistorycombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
